bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/editor/targetComponent/TelescopePosTableWidget.java

```diff
@@ -315,7 +315,7 @@ private static Icon bagsIcon(boolean enabled) {
                             final Option<AgsGuideQuality> quality = guideQuality(ags, guideProbe, target);
                             final boolean enabled = isPrimaryGroup && primary.exists(target::equals);
 
-                            final Row row = new GuideTargetRow(isActive, quality, enabled, guideProbe, tup._2(), target, baseCoords, when);
+                            final Row row = new GuideTargetRow(isActive, quality, enabled, guideProbe, tup._2() + 1, target, baseCoords, when);
```

Comment:
Oh yeah. There is definitely room for significant cleaning and refactoring here.
Since this will be changing with BAGS, I'd rather make a note to do the refactoring then instead of here in this PR.